configuration/security.md

```diff
@@ -180,6 +217,24 @@ To disable the firewall, and disable start up on boot, use:
 sudo ufw disable
 ```
 
+To disable logging, use:
```

Comment:
Is it worth explaining why a user might want to / not want to disable logging?

Reply:
> Is it worth explaining why a user might want to / not want to disable logging?

Good point.

Reply:
The new text suggests _when_ you might want to disable logging, but it still doesn't suggest _why_ :wink:

Reply:
Alright, changing it now.

Reply:
"you may wish to disable logging as you'll be denying quite a lot of requests"

Still not clear (to me) _why_ this means you'd want to disable logging? Does logging slow down ufw? Or does it use a lot of diskspace? Or something else?